### PR TITLE
Add "OR" operator to query builder

### PR DESCRIPTION
**Pull request by dzubchik**
Recently I faced with problem of building complex query and I should do this using query builder. I have not found possibilities to do this, so this is pull-request that does not break compatibility.

### CODE REVIEW

File: phalcon/mvc/model/query/builderinterface.zep [hunk 1]
```diff
@@ -26,6 +26,10 @@ namespace Phalcon\Mvc\Model\Query;
  */
 interface BuilderInterface
 {
+    const OPERATOR_OR = "or";
```
Comment (Green-Cat) [inline phalcon/mvc/model/query/builderinterface.zep]: The indentation is a bit off here.

**Comment by sergeyklay** [comment]
For indents we use tabs in zep files instead of spaces

**Comment by dzubchik** [comment]
@sergeyklay , just a second and I will correct it. I think this possibility should be highlighted in documentation. How can I do this?

**Comment by sergeyklay** [comment]
Looks good to me.

@andresgutierrez

**Comment by xboston** [comment]
:+1:

**Comment by sergeyklay** [comment]
Thank you very much, man

**Pull request merged**